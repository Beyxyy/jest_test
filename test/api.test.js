const api= require('../api/api.js');

test("Le rÃ©sultat de l'API doit correspondre au snapshot", () => {
    const result = api.ResultAPI();
    console.log(result);
    expect(result).toMatchSnapshot();
});



