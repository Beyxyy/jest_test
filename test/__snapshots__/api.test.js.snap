// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Le r√©sultat de l'API doit correspondre au snapshot 1`] = `
{
  "age": "20",
  "email": "randomuser@gmail.com",
  "firstname": "Random",
  "friends": [
    {
      "id": [
        "2",
        "5",
        "7",
        "9",
        "10",
      ],
    },
  ],
  "hobbys": [
    {
      "sport": [
        "football",
        "gym",
      ],
    },
    {
      "music": [
        "rock",
        "pop",
        "drill",
      ],
    },
    {
      "food": [
        "pizza",
        "chicken",
      ],
    },
  ],
  "id": "1",
  "lastname": "User",
}
`;
